languages/elixir/exercises/concept/booleans/.docs/after.md

```diff
@@ -0,0 +1,29 @@
+Elixir represents true and false values with the boolean type. There are only two values -- _true_ and _false_. These values can be bound to a variable:
+
+```elixir
+is_true = true
+is_false = false
+```
+
+We can combine boolean values into expressions using boolean logical operators.  `and/2`, `or/2`, and `not/1` are commonly used in expressions to evaluate multiple boolean values to a single result. These are considered strict operators, since they only work with boolean values.
```

Comment:
I don't know if this is useful to mention, but you could consider referring to operator precedence rules and how to "escape" them (see [this example](https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/booleans/.docs/after.md)).